.gitattributes

```diff
@@ -0,0 +1,7 @@
+# Ignore all test and documentation for archive
```

Comment:
Hey, I don't have any real arguments against these changes. If it helps to make exported archive clearer, let's do it. My only point is: you say here "Ignore documentation...", but I don't see any ignored docs. Should we ignore README.md as well? Is it a good practise to ignore the docs?